src/app_service/service/community/service.nim

```diff
@@ -262,6 +262,7 @@ QtObject:
       historyArchiveDownloadTaskCommunityIds*: HashSet[string]
       communityMetrics: Table[string, CommunityMetricsDto]
       communityInfoRequests: Table[string, Time]
+      sharedAddresses: seq[string]
```

Comment:
This doesn't really make sense. Each community has its own shared addresses. We have a function that retrieves the shared accounts for the user for a given community. It's called `asyncGetRevealedAccountsForMember` (it gets it from the DB)